packages/bpk-docs/src/pages/NativeCardsPage/NativeCardsPage.js

```diff
@@ -104,6 +106,29 @@ const components = [
       },
     ],
   },
+  {
+    id: 'large-corner-style',
+    title: 'Large corner style',
+    blurb: 'Cards can have a `large` corner style.',
```

Comment:
```suggestion
    blurb: 'Cards can have more rounded corners when appropriate.',
```
I feel that the blurb as-is doesn't say anything that isn't already addressed in the title.